index.html

```diff
@@ -95,7 +100,8 @@ <h3 class="title-subheader">Aug 23-26  //  Portland, OR </h3>
           <a href="#sponsors" class="link white"><li>Sponsors</li></a>
           <a href="#code-of-conduct" class="link white"><li>Code of Conduct</li></a>
           <a href="#faq" class="link white"><li>FAQ's</li></a>
-          <a href="https://ti.to/write-speak-code/2017-conference/" class="register-now link white" target="_blank"><li>Register Now</li></a>
+          <!-- <a href="https://ti.to/write-speak-code/2017-conference/" class="register-now link white" target="_blank"><li>Register Now</li></a> -->
+          <li id="nav-register"><tito-button event="write-speak-code/2017-conference" class="register-now link white">Register Now</tito-button></li>						
         </ul>
```

Comment:
After removing that id and classes you could then remove the following selections from custom to keep the css clean:
```css
.nav-section .register-now {
  color: #EB6851;
}
```

```css
.nav-section a.register-now li:hover {
  background-color: #EB6851;
  color: white;
}
```


```css
#qcPriNav #nav-register a {
  color: #EB6851;
}
```